static/js/components/SourceTable.jsx

```diff
@@ -334,9 +356,49 @@ const RenderShowClassification = ({ source }) => {
     });
   };
 
+  const addVotes = (vote) => {
+    source.classifications?.forEach((c) => {
+      dispatch(sourceActions.addClassificationVote(c.id, { vote })).then(
+        (result) => {
+          if (result.status === "success") {
+            dispatch(showNotification("Vote registered"));
```

Comment:
Displaying "Vote registered" for each classification is accurate but can temporarily take up a lot of screen space. Is it worth adding an array to keep track of each "success" and, if votes are uniformly successful, displaying a single "Votes registered" notification?

Reply:
@bfhealy I pushed a change for this.